### ISSUE TITLE
External url cannot be accessed from other devices

### ISSUE THREAD

**Issue by 919Studios** [BrowserSync/browser-sync#1792]
### Issue details

**tldr; external url cannot be accessed from other devices.**
Any help is much appreciated! - I can view the external url from my development machine but not from any other devices on the same wifi network. I have tried several configuration modifications but none have solved this. I appreciate any help on this!

### Steps to reproduce/test case

just a standard install…

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [ 2.26.7   ]
- Node        [ 12.16.2   ]
- Npm         [  6.14.4  ]

### Affected platforms

- [ ] linux
- [ ] windows
- [x] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [ ] Gulp
- [ ] Grunt
- [x] CLI

### If CLI, please paste the entire command below

npm run dev
